devel/MacOSX.sdk/Portfile

```diff
@@ -17,7 +17,7 @@ switch ${os.arch} {
     arm64 -
     i386 -
     default {
-        if {${build_arch} eq "i386"} {
+        if {${configure.build_arch} eq "i386"} {
```

Comment:
I can never remember the difference.

Reply:
My understanding is that `${build_arch}` is what is set in macports.conf.
`${configure.build_arch}` is the value after [consistency checks are implemented ](https://github.com/macports/macports-base/blob/3f194103225289dbef2131afc2548a7d08d8eb38/src/port1.0/portconfigure.tcl#L303C50-L303C72).